### ISSUE TITLE
Prototype export/import in proposed zipped Tale format

### ISSUE THREAD

**Issue by craig-willis** [whole-tale/girder_wholetale#224]
Prototype Tale export/import using the proposed zip format https://github.com/whole-tale/tale_serialization_formats

* Update the existing export endpoint to create ZIP archive
  * Workspace directory
  * Generate manifest.json
  * Copy default README.md (we need to create this)
* Create new /tale/import endpoint
  * Upload zip file ala https://girder.wholetale.org/api/v1#!/file/file_readChunk
  * Unzips
  * Reads manifest.json
  * Creates Tale object
  * Copies files to Workspace
 
Assume this is happening with the r2d support -- default to an existing image.

**Comment by ThomasThelen** [comment]
I implemented Tale exporting that supports files in the workspace along with files that are externally referenced. This can be found in the `tale_Export` branch in girder_wholetale.

Importing was implemented in a separate branch, `tale_zip_import` which was forked from `tale_Export`. The import functionality currently supports external datasets and not files in the workspace, which should be some delta amount of effort. When importing, you'll want to check out the `tale_export_import` branch in gwvolman.

How to use this:
   1. Use `https://girder.local.wholetale.org/api/v1#!/tale/tale_exportTale` to export a Tale to disk
   2. Upload the zip to your home directory
   3. Get the itemId of the zip directory
   4. Use `https://girder.local.wholetale.org/api/v1#!/tale/tale_import_zip` with the itemId to process the zipped Tale
